changelog.md

```diff
@@ -4,6 +4,8 @@
 We removed deprecated Kamon based metrics from the code base (see the 1.7.xxx changelog for details on new metrics). This led to removal of deprecated command line arguments e.g. old reporters like `--reporter_graphite`, `--reporter_datadog`, `--reporter_datadog` and `--metrics_averaging_window`.
 ### Apps names restrictions (breaking change)
 From now on, apps which uses ids which ends with "restart", "tasks", "versions" won't be valid anymore. Such apps already had broken behavior (for example it wasn't possible to use a `GET /v2/apps` endpoint with them), so we made that constraint more explicit. Existing apps with such names will continue working, however all operations on them (except deletion) will result in an error. Please take care of renaming them before upgrading Marathon.
+### Standby marathon instances no longer proxy events
+We no longer allow a standby Marathon instance to proxy /v2/events from Marathon master.
```

Comment:
I would maybe add more context. Maybe name of that feature flag or something? I would not be probably able to tell what this means exactly for me as a user